### PR TITLE
2017 hero section

### PR DESCRIPTION
**Pull request by darrendub**
## Purpose
Adding styles, content, and layout to johnsonfamilycookieparty.com to reflect event details and information. 

## Notes
• Since this will not be affecting index.html—only affect stranger.html which isn't linked—I think we can safely merge to master so we can start sharing this with the event planners to get copy and layout approval
• Looking for guidance on responsive styles but that can be in a follow up PR

screencapture-file-users-dwong-public-johnsonfamilycookieparty-stranger-html-1513324394474

_users_dwong_public_johnsonfamilycookieparty_stranger html iphone 6 plus

### CODE REVIEW

File: assets/stylesheets/_sections.scss [hunk 1]
```diff
@@ -21,15 +26,21 @@
...
   padding: 6em 0;
   text-align: center;
+
 }
 
 .hero-section__logo {
   margin: 0 auto;
   margin-bottom: 4em;
+  width:100%;
+  max-width: 600px;
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss]: 🤔 is this implementation-specific? if not, would it make sense to extract this to `$width--med`? (`$wrapper--med`, etc…?)

File: assets/stylesheets/_sections.scss [hunk 2]
```diff
@@ -57,13 +68,18 @@
         linear-gradient(rgba(black, 0.5), rgba(black, 0.5)),
         black
         url("#{$asset-path}" + "#{$joyce-background}");
-  background-position: center bottom;
+  background-position: bottom;
   background-repeat: no-repeat;
-  background-size: auto 150%;
+  @media screen and (min-width: $bp-small){
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss]: what's `$bp`?

Comment (emilypstewart) [reply assets/stylesheets/_sections.scss]: oh, breakpoint. derp.

File: assets/stylesheets/_sections.scss [hunk 3]
```diff
@@ -88,16 +103,31 @@
...
 .cookie-decorator {
   fill: $ruby;
   height: 5em;
   padding: 0 1em;
   width: 8em; }
 
-.leaderboard {
+.leaderboard { // Need to rearrange grid for small screens
+  padding-bottom: 3em;
+  display: grid;
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss]: Look at you!!

File: assets/stylesheets/_sections.scss [hunk 4]
```diff
@@ -110,61 +140,160 @@
...
+  }
+}
+
+
+.blink{
+  color:$lemon;
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss]: Nice! Might make sense to separate the keyframes into an animations partial? (unless that seems overkill?)

File: assets/stylesheets/_sections.scss [hunk 5]
```diff
@@ -110,61 +140,160 @@
...
   font-size: $type--small;
   color: $white;
+  text-align: center;
 }
 
-.events-section {
-  align-items: center;
-  display: flex;
+.events-section{
+  background-color: #020710;
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss]: (i see this raw hex 😜)

File: assets/stylesheets/_sections.scss [hunk 6]
```diff
@@ -110,61 +140,160 @@
...
+.events-section__costume {
+  background: 
+    black
+    url("#{$asset-path}" + "#{$costume-background}");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss:202]: the inline media queries are interesting. 🤔 I know @miketlawrence advocates for this…I'm still on the fence, but it works here.

File: assets/stylesheets/_sections.scss [hunk 7]
```diff
@@ -110,61 +140,160 @@
...
 .adios-section__headline {
   font-size: $type--xl;
 }
+
+.logo--small {
+  width: 100%;
+  max-width: 200px;
+}
+
+.upside-up {
```
Comment (emilypstewart) [inline assets/stylesheets/_sections.scss:270]: hahah, only on a Stranger Things project would this name make sense

**Review by emilypstewart — APPROVED** [review]
This is awesome! thanks for the clean commit messages; it makes it so much easier to review. Added a few suggestions…but nicely done 🙌

File: assets/stylesheets/_sections.scss (hunk 2 above)
Comment (darrendub) [reply assets/stylesheets/_sections.scss]: derrrrpy (i totally lifted that variable name from the internet)

File: assets/stylesheets/_sections.scss (hunk 5 above)
Comment (darrendub) [reply assets/stylesheets/_sections.scss]: oh ya hahah i should probably update that lol $variables-only

File: assets/stylesheets/_sections.scss (hunk 6 above)
Comment (darrendub) [reply assets/stylesheets/_sections.scss:202]: i find them easier to pinpoint when im trying to affect the responsive style of a certain class

File: assets/stylesheets/_sections.scss (hunk 4 above)
Comment (darrendub) [reply assets/stylesheets/_sections.scss]: yes maam

File: assets/stylesheets/_sections.scss (hunk 7 above)
Comment (darrendub) [reply assets/stylesheets/_sections.scss:270]: lol

**Pull request merged**